controllers/greptimedbcluster/deployers/datanode.go

```diff
@@ -411,16 +354,6 @@ func (d *DatanodeDeployer) generatePodTemplateSpec(cluster *v1alpha1.GreptimeDBC
 			Protocol:      corev1.ProtocolTCP,
 			ContainerPort: cluster.Spec.PostgresServicePort,
 		},
```

Comment:
suggestion: The service not define the postgres port, pod template spec not need to be defined[here](https://github.com/GreptimeTeam/greptimedb-operator/blob/2167a0996ced882d80e2bbb8eb8130b888178014/controllers/greptimedbcluster/deployers/datanode.go#L353)